TemporaryOysters/bangazon_website
# Able to view product detail from products page

## Pull Requests



## Descriptions:

**Please describe what issue this PR solves**

- Now able to click on a product from the Products page which will take the user to the detail page.


**What's next?**

- Get 'Buy Product' button working correctly. 






## Steps to Test:


1) git fetch
2) git checkout rvs-product-detail
4) delete DB and migrations
5) python manage.py makemigrations
6) python manage.py migrate
7) python manage.py loaddata banagzon_website/load_data.json
8) python manage.py runserver

Make sure that products display and you are able to click on a product name. That should take you to the product detail view.
